dev/bin/report-portal/report-portal-import-from-workflows.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+set -e
+
+# Import test result from Github workflow to Report Portal.
+#
+# Usage: report-portal-import-from-workflows.sh repo_owner, repository, workflow, launch_name
+# Expected environment variables
+# GITHUB_TOKEN ... token with access to workflow run artifacts
+# REPORT_PORTAL_URL ... Report Portal URL
+# REPORT_PORTAL_TOKEN ... Report Portal token
+# REPORT_PORTAL_PROJECT ... Report Portal project
+
+check_env_variable_exists() {
+    env_variable_name=$1
+    if [[ -z "${!env_variable_name}" ]]; then
+        echo "Error: $env_variable_name environmental variable must be set" 1>&2;
+        exit 1
+    fi
+}
+
+import_junit() {
```

Comment:
Is there any specific reason why not to use Data Router for importing the junit results? 

I think it is totally fine to start what is implemented here, but the DR is supposed to help us to avoid implementing the import logic by ourselves, so it might be a good practice in future?

Reply:
Yes, I agree with improving to Data Router in the future. It should not be hard if it can import junits.